proposals/bring_your_own_argo_workflows/byoaw_test_plan.md

```diff
@@ -0,0 +1,305 @@
+# Test Plan: Bring Your Own Argo Workflows (BYOAW)
```

Comment:
FWIW I think the common acronym being thrown around is BYOAWF (Bring Your Own Argo WorkFlows)